package.json

```diff
@@ -40,6 +40,7 @@
     "npm-run-all": "^4.1.5",
     "remark-cli": "^7.0.1",
     "remark-toc": "^6.0.0",
+    "tsconfigs": "^4.0.2",
```

Comment:
https://www.npmjs.com/package/tsconfigs brought to you by some random developer. He probably has good intentions.